docusaurus/android_versioned_docs/version-5.x.x/04-compose/05-message-components/05-attachments-picker.mdx

```diff
@@ -147,6 +147,7 @@ class AttachmentsPickerCustomTabFactory: AttachmentsPickerTabFactory {
 
     @Composable
     override fun pickerTabContent(
+        onAttachmentPickerAction: (AttachmentPickerAction) -> Unit,
```

Comment:
Should we change the docs for v5 if this feature will be available in v6 only?

Reply:
That's a great point! Fixed: [57354ef](https://github.com/GetStream/stream-chat-android/pull/5305/commits/57354efde216db50a3fc4a4ec85cce83f73fb0ec)